conf/proxy.conf

```
@@ -277,6 +277,23 @@ tokenAudienceClaim=
 # The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token, need contains this.
 tokenAudience=
 
+### --- SASL Authentication Provider --- ###
+
+# This is a regexp, which limits the range of possible ids which can connect to the Broker using SASL.
+# Default value: `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*",
+# so only clients whose id contains 'pulsar' are allowed to connect.
+saslJaasClientAllowedIds=
```

If there is a default, I think we should specify it here. Will it work to do the following?
```suggestion
saslJaasClientAllowedIds=.*pulsar.*
```